scipy/stats/_axis_nan_policy.py

```diff
@@ -226,6 +234,40 @@ def _check_empty_inputs(samples, axis):
     return output
 
 
+def _expand_dims(arr, axes):
+    """
+    Insert new axes that will appear at the `axis` position in the expanded
+    array shape.
+    """
+    # Since NumPy 1.18, `expand_dims` supports tuple axis. As SciPy
+    # supports any NumPy >= 1.17, we can't guarantee support for tuple
+    # axis by NumPy. This for loop can be removed when support for
+    # NumPy 1.17 is dropped.
```

Comment:
Actually (this is on main) SciPy 1.9 will only supports Python 3.8 hence NumPy >=1.18.5 😃 So we can already remove this.

Reply:
That's great! Will update this.

Reply:
Yeah it seems to happen a lot recently. Does not mean it's this PR.